<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgRotateImage</class>
 <widget class="QWidget" name="WdgRotateImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotate Image</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpDirection">
     <property name="title">
      <string>Orientation</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioCW">
        <property name="text">
         <string>&amp;Right</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioCCW">
        <property name="text">
         <string>&amp;Left</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpAngle">
     <property name="title">
      <string>Angle</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radio90">
        <property name="text">
         <string>90 &amp;degrees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio180">
        <property name="text">
         <string>&amp;180 degrees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio270">
        <property name="text">
         <string>270 de&amp;grees</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioCustom">
          <property name="text">
           <string>&amp;Custom:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KisAngleSelector" name="angleSelectorCustom" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="spacer4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisAngleSelector</class>
   <extends>QWidget</extends>
   <header location="global">KisAngleSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
